lib/tasks/db.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+if Rake::Task['db:schema:dump:clickhouse'].present?
+  Rake::Task['db:schema:dump:clickhouse'].enhance do
+    Rake::Task['db:clickhouse:filter'].invoke
+  end
+end
+
+namespace :db do |args|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `args`. You can omit the argument if you don't care about it. (https://rubystyle.guide#underscore-unused-vars)